.gitignore

```diff
@@ -42,5 +42,9 @@ nosetests.xml
 # Node
 node_modules
 
+DjangoExpenses/
```

Comment:
Those `.gitignore` changes are not relevant to `django-expenses`, please undo them.